### PR TITLE
[i18n] Change Play button behavior to navigate to a translated URL /en/room/:id or /pt/room/:id (part. 3)

### PR DESCRIPTION
**Pull request by richardaum**
### Issue

Clicking on Play on GameModePicker leads to an untranslated URL /room/:id

### Goal
Change behavior to navigate to translated URL /en/room/:id or /pt/room/:id

### Preview

Where is it located?

https://user-images.githubusercontent.com/2018161/116758287-1aac3e00-a9e6-11eb-8af7-f305a4b558c7.mp4

### CODE REVIEW

File: web/src/infra/gameInfo/GameInfo.tsx [hunk 1]
```diff
@@ -96,4 +95,4 @@ class GameInfo extends React.Component<gameInfoProps, {}> {
   }
 }
 
-export default withRouter(GameInfo as any);
```
Comment (richardaum) [inline web/src/infra/gameInfo/GameInfo.tsx:99]: Important to mention that no properties from Router were been used on this component, so I decided to remove the HOC.

**Comment by vdfdev** [comment]
LGTM, worked with the flag disabled and enabled

**Review by vdfdev — APPROVED** [review]

**Comment by richardaum** [comment]
Can we merge it?

**Comment by vdfdev** [comment]
sorry, was waiting for the CI and forgot :P

**Pull request merged**